src/Dfe.PlanTech.Application/Persistence/Commands/WebhookMessageProcessor.cs

```diff
@@ -18,7 +18,7 @@ public async Task<IServiceBusResult> ProcessMessage(string subject, string body,
         {
             var payload = MapMessageToPayload(body);
 
-            await cache.InvalidateCacheAsync(payload.Sys.Id);
+            cache.InvalidateCacheAsync(payload.Sys.Id);
```

Comment:
I get a linting error on this, is there something else you need to do around this being a non awaited task?